### PR TITLE
Translation of Git episode 6

### LINKED ISSUE #66: Translation: git-novice Episode 6
Translation: git-novice Section 6 "Ignoring Things"
-----

* Please assign yourself (or add a comment) to the issue **before** you start translating, so other people know that you are translating.
* Once you have finished translating, send a pull request and reference this issue.

翻訳：git-novice セクション6 "Ignoring Things"
-----

* あなたが翻訳し始めたと分かるように、翻訳を**始める前**にこのイシューを自分に割り振って(assign)、あるいはコメントを書いて下さい。
* 翻訳が終わったら、プルリクエストを開いて、このイシューを参照・リンクして下さい。

https://github.com/swcarpentry-ja/i18n/blob/b3225373bbf9b2f14b8882e4bc019cbd9dbad114/po/git-novice.ja.po#L4670

### PR DESCRIPTION
**Pull request by rikutakei**
There may be a few words missing in the translation, as I thought some of these words were unnecessary/redundant in the Japanese translation, but please suggest better alternatives if you think of any.

よろしくお願いします！

### CODE REVIEW

File: po/git-novice.ja.po [hunk 1]
```diff
@@ -4675,16 +4675,16 @@ msgid ""
 "---"
 msgstr ""
 "---\n"
-"title: Ignoring Things\n"
+"title: ファイルを無視する\n"
 "teaching: 5\n"
 "exercises: 0\n"
 "questions:\n"
-"- \"How can I tell Git to ignore files I don't want to track?\"\n"
+"- \"Git で追跡したくないファイルを指定するにはどうすればいいの？\"\n"
```
Comment (joelnitta) [inline po/git-novice.ja.po]: Suggestions for `questions`, `objectives`, and `keypoints` are to make style consistent with [R gapminder translation](https://swcarpentry-ja.github.io/r-novice-gapminder/ja/)
```suggestion
"- \"Git で追跡したくないファイルを指定するにはどうすればよいですか？\"\n"
```

File: po/git-novice.ja.po [hunk 2]
```diff
@@ -4675,16 +4675,16 @@ msgid ""
...
 "objectives:\n"
-"- \"Configure Git to ignore specific files.\"\n"
-"- \"Explain why ignoring files can be useful.\"\n"
+"- \"Git で追跡したくないファイルを指定する\"\n"
```
Comment (joelnitta) [inline po/git-novice.ja.po]:
```suggestion
"- \"Git で追跡したくないファイルを指定しましょう\"\n"
```

File: po/git-novice.ja.po [hunk 3]
```diff
@@ -4675,16 +4675,16 @@ msgid ""
...
+"- \"ファイルを無視する利点を理解する\"\n"
```
Comment (joelnitta) [inline po/git-novice.ja.po]:
```suggestion
"- \"ファイルを無視する利点を理解しましょう\"\n"
```

File: po/git-novice.ja.po [hunk 4]
```diff
@@ -4675,16 +4675,16 @@ msgid ""
...
 "keypoints:\n"
-"- \"The `.gitignore` file tells Git what files to ignore.\"\n"
+"- \"`.gitignore` で無視するファイルを指定します\"\n"
```
Comment (joelnitta) [inline po/git-novice.ja.po]:
```suggestion
"- \"`.gitignore` で無視するファイルを指定する\"\n"
```

File: po/git-novice.ja.po [hunk 5]
```diff
@@ -4694,10 +4694,10 @@ msgid ""
 "or intermediate files created during data analysis?\n"
 "Let's create a few dummy files:"
 msgstr ""
-"What if we have files that we do not want Git to track for us,\n"
-"like backup files created by our editor\n"
-"or intermediate files created during data analysis?\n"
-"Let's create a few dummy files:"
+"Git に追跡して欲しくないファイル、例えばエディタが作成するバックアップファイルや\n"
```
Comment (joelnitta) [inline po/git-novice.ja.po]:
```suggestion
"Git に追跡して欲しくないファイル、例えばエディタが作成したバックアップファイルや\n"
```

File: po/git-novice.ja.po [hunk 6]
```diff
@@ -4694,10 +4694,10 @@ msgid ""
...
+"データ解析中に作られる中間ファイルなどは、\n"
```
Comment (joelnitta) [inline po/git-novice.ja.po]:
```suggestion
"データ解析中に作られた中間ファイルなどは、\n"
```

File: po/git-novice.ja.po [hunk 7]
```diff
@@ -4798,18 +4798,18 @@ msgid ""
 "(If any of these files were already being tracked,\n"
 "Git would continue to track them.)"
 msgstr ""
-"These patterns tell Git to ignore any file whose name ends in `.dat`\n"
-"and everything in the `results` directory.\n"
-"(If any of these files were already being tracked,\n"
-"Git would continue to track them.)"
+"入力したパターンは、 Git に `.dat` で終わるファイル名と\n"
```
Comment (joelnitta) [inline po/git-novice.ja.po:4801]: This is a comment about the original, but I think "pattern" / パターン might be difficult for novices to understand.

**Review by joelnitta — COMMENTED** [review]
(review in progress, saving what I've done so far)

File: po/git-novice.ja.po [hunk 8]
```diff
@@ -4798,18 +4798,18 @@ msgid ""
...
+"引き続き追跡されます。）"
 
 #: git-novice/_episodes/06-ignore.md:69
 msgid ""
 "Once we have created this file,\n"
 "the output of `git status` is much cleaner:"
 msgstr ""
-"Once we have created this file,\n"
-"the output of `git status` is much cleaner:"
+"このファイルを作った後に\n"
```
Comment (joelnitta) [inline po/git-novice.ja.po]:
```suggestion
"このファイルを作った後\n"
```

File: po/git-novice.ja.po [hunk 9]
```diff
@@ -4842,11 +4842,11 @@ msgid ""
...
 msgstr ""
-"The only thing Git notices now is the newly-created `.gitignore` file.\n"
-"You might think we wouldn't want to track it,\n"
-"but everyone we're sharing our repository with will probably want to ignore\n"
-"the same things that we're ignoring.\n"
-"Let's add and commit `.gitignore`:"
+"Git は新しく作られた `.gitignore` ファイルしか表示していません。\n"
+"このファイルは追跡しなくても良いかと思うでしょうが、\n"
+"リポジトリを共有する際に、他の人達も私達が無視したものを\n"
+"同じように無視すると思うので、\n"
```
Comment (joelnitta) [inline po/git-novice.ja.po]: I think this is closer to the original
```suggestion
"同じように無視したいでしょうから、\n"
```

File: po/git-novice.ja.po [hunk 10]
```diff
@@ -4881,8 +4881,8 @@ msgid ""
 "As a bonus, using `.gitignore` helps us avoid accidentally adding to the "
 "repository files that we don't want to track:"
 msgstr ""
-"As a bonus, using `.gitignore` helps us avoid accidentally adding to the "
-"repository files that we don't want to track:"
+"`.gitignore` を作った事によって、間違えて不要なファイルを"
+"リポジトリに追加する事もなくなります："
```
Comment (joelnitta) [inline po/git-novice.ja.po]:
```suggestion
"リポジトリに追加する事を防ぐことができます："
```

File: po/git-novice.ja.po [hunk 11]
```diff
@@ -5002,41 +5002,41 @@ msgid ""
...
+"> > 解決できます：\n"
 "> >\n"
 "> > `results/plots/`\n"
 "> >\n"
-"> > If, instead, you want to ignore everything in `/results/`, but wanted to "
-"track\n"
-"> > `results/data`, then you can add `results/` to your .gitignore\n"
-"> > and create an exception for the `results/data/` folder.\n"
-"> > The next challenge will cover this type of solution.\n"
+"> > もし、`/results/` 内のファイルを全て"
```
Comment (joelnitta) [inline po/git-novice.ja.po]:
```suggestion
"> > その代わりに、もし、`/results/` 内のファイルを全て"
```

File: po/git-novice.ja.po [hunk 12]
```diff
@@ -5002,41 +5002,41 @@ msgid ""
...
+"無視して、\n"
+"> > `results/data` のみを追跡したいのであれば、`results/` を .gitignore\n"
+"> > に加えて、`results/data/` ファイルを対象外にすれば問題ありません。\n"
```
Comment (joelnitta) [inline po/git-novice.ja.po]:
```suggestion
"> > に加えて、`results/data/` ファイルを対象外にすることができます。\n"
```

File: po/git-novice.ja.po [hunk 13]
```diff
@@ -5058,26 +5058,26 @@ msgid ""
 "> > The exclamation point operator will include a previously excluded entry."
 msgstr ""
 ">\n"
-"> How would you ignore all `.data` files in your root directory except for\n"
-"> `final.data`?\n"
-"> Hint: Find out what `!` (the exclamation point operator) does\n"
+"> ルートディレクトリ内にある全ての `.data` ファイルの中から、\n"
```
Comment (joelnitta) [inline po/git-novice.ja.po]:
```suggestion
"> `final.data`以外の、\n"
```

File: po/git-novice.ja.po [hunk 14]
```diff
@@ -5058,26 +5058,26 @@ msgid ""
...
+"> `final.data` のみを含めるにはどうすればいいのでしょう？\n"
```
Comment (joelnitta) [inline po/git-novice.ja.po]:
```suggestion
"> ルートディレクトリ内にある他の `.data` ファイルを全て無視したい場合はどうすればいいのでしょう？\n"
```

File: po/git-novice.ja.po [hunk 15]
```diff
@@ -5002,41 +5002,41 @@ msgid ""
...
+"> > たまに `**` パターンが役に立ちます。このパターンを使えば、\n"
+"> > 複数のディレクトリに対象を適用できます。例えば、`**/results/plots/*` を使えば "
+"Git に\n"
+"> > ルートディレクトリ内にある全てのディレクトリ内の `results/plots` ディレクトリを無視することができます。"
 
 # blockquote, which can be cascaded
 #: git-novice/_episodes/06-ignore.md:177
 msgid "> ## Including Specific Files"
-msgstr "> ## Including Specific Files"
+msgstr "> ## 特定のファイルを含める"
```
Comment (joelnitta) [inline po/git-novice.ja.po]: I feel like 含める doesn't quite work here... there's no sense of where it's being included. The idea is to "include" files so they get tracked, so I think 追跡する could work.
```suggestion
msgstr "> ## 特定のファイルを追跡する"
```

File: po/git-novice.ja.po [hunk 16]
```diff
@@ -5203,27 +5203,27 @@ msgid ""
...
 ">\n"
-"> 3. You find that the file `log_01` is very important after all, add it to "
-"the tracked files without changing the `.gitignore` again.\n"
+"> 3. 最終的に `log_01` ファイルがものすごく重要であることが分かりました。"
+"`.gitignore` を編集せずに、このファイルを追跡しているファイルに加えてください。\n"
 ">\n"
-"> 4. Discuss with your neighbor what other types of files could reside in "
-"your directory that you do not want to track and thus would exclude via `."
-"gitignore`.\n"
+"> 4. 隣の人と、他にどのファイルを無視するのかを決めて、"
```
Comment (joelnitta) [inline po/git-novice.ja.po]:
```suggestion
"> 4. 隣の人と、追跡したくないファイルは他にどのようなものがあるのか、そして"
```

**Review by joelnitta — COMMENTED** [review]
Finished the review! Sorry for the delay.

File: po/git-novice.ja.po (hunk 7 above)
Comment (rikutakei) [reply po/git-novice.ja.po:4801]: Theoretically people would have done bash/shell lessons first before the git lessons, so they should be able to understand it

**Review by joelnitta — APPROVED** [review]

**Pull request merged**